.github/ISSUE_TEMPLATE/new_practice.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Practice suggestion
+about: Suggest new article about a good Practice 
+title: "[new_practice]"
+labels: 'article_type: practice'
```

Comment:
maybe to add label "suggestion"?

Reply:
What about the existing label enhancement?